airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1964,13 +1972,14 @@ components:
         __type: {type: string}
```

Comment:
Should add other fields to required also?

Reply:
Done